skyrim-platform/src/platform_se/skyrim_platform/PapyrusTESEvents.cpp

```diff
@@ -0,0 +1,17 @@
+#include "PapyrusTESEvents.h"
+
+namespace PapyrusTESEvents {
+
+TESSpellCastEvent::TESSpellCastEvent()
+  : TESSpellCastEvent(nullptr, static_cast<RE::FormID>(0))
+{
+}
+
+TESSpellCastEvent::TESSpellCastEvent(RE::TESObjectREFR* a_caster,
```

Comment:
a_caster -> cater, a_spell -> spell

We strictly follow this guide in new code https://github.com/skyrim-multiplayer/skymp/blob/main/docs/docs_cpp_code_guidelines.md